src/components/playground/TypographyInput.js

```diff
@@ -224,15 +224,11 @@ function FontFamily(props) {
 
 function TypographyInput(props) {
   const { path, label } = props;
-  const { getPropByPath, setPropByPath } = usePlaygroundUtils();
-  const initialValue = getPropByPath(path);
-  const [value, setValue] = useState(initialValue);
+  const { getPropByPath } = usePlaygroundTheme();
+  const { setPropByPath } = usePlaygroundUtils();
+  const [value, setValue] = useState(() => getPropByPath(path));
   const [isValid, setValid] = useState(true);
-  const [defaultValue] = useState(initialValue);
-
-  useEffect(() => {
-    setValue(initialValue);
-  }, [initialValue]);
+  const defaultValue = useDefaultValue(value);
 
   const resetTypography = () => {
     setPropByPath(path, defaultValue);
```

Comment:
@RubenSmn all of the resetTypography's need to have the .current added to defaultValue or they dont work to reset. There are 3 of them. like this
``` 
 const resetTypography = () => {
    setValue(defaultValue.current);
    setPropByPath(path, defaultValue.current);
  };
```

awesome job on all of this so far though, have a few more individual commits to look at.

Reply:
Thanks I did a lot of testing on the `Color` side of the components. But forgot the `Typography` resets! This gives me the idea that the `.current` is weird to have since the hooks should just return the `value` :smile: .